src/components/Person.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Person.scss';
+
+const Person = ({ name, age, sex, partnerName, isMarried }) => ((
+  <div className="App">
+    <section className="Person">
+      <h2 className="Person__name">
+        My name is
+        {' '}
+        {name}
+      </h2>
+      <p className="Person__age">
+        {age && (`I am ${age}`)}
+      </p>
+      <p className="Person__partner">
+        {!isMarried ? 'I am not married'
+          : `My
+        ${isMarried && ((sex === 'm' && 'wife')
```

Comment:
why do you need to check isMarried here too? you've already checked it in ternary